src/service/owners-impl.js

```diff
@@ -30,6 +59,17 @@ export class Owners {
    */
   scheduleUnlayout(parentElement, subElements) {}
 
+  /**
+   * Schedules preload for the specified sub-elements that are children of the
+   * parent element. The parent element may choose to send this signal either
+   * because it's an owner (see {@link setOwner}) or because it wants the
+   * preloads to be done sooner. In either case, both parent's and children's
+   * priority is observed when scheduling this work.
+   * @param {!Element} parentElement
+   * @param {!Element|!Array<!Element>} subElements
+   */
+  schedulePreload(parentElement, subElements) {}
+
   /**
    * A parent resource, especially in when it's an owner (see {@link setOwner}),
```

Comment:
nit: move setOwner() to the first method

Reply:
That method is nonexistent yet.